### ISSUE TITLE
Web facet is not generated.

### ISSUE THREAD

**Issue by jehrhardt** [mpeltonen/sbt-idea#154]
I have taken the [scalatra sample](https://github.com/scalatra/scalatra-sbt-prototype) and run _sbt gen-idea_ (sbt version 0.11.3). The resulting project does not contain a web facet and an artifact, that can be deployed to a Tomcat or Jetty inside of IDEA Ultimate Edition, is also not created.

Btw, when a web facet would be generated, it would [have a deploymentDescriptor configured](https://github.com/mpeltonen/sbt-idea/blob/sbt-0.11/src/main/scala/org/sbtidea/IdeaModuleDescriptor.scala#L148). Since Servlet 3.0 a web.xml is not required anymore, thus the configuration should be optional.

**Comment by mpeltonen** [comment]
Yes, web facet support is not implemented yet (or re-implemented for the sbt 0.10+ version of the plugin. It was in 0.7 versions).

Pull requests are always welcomed :)
